core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/seo/components/dot-device-selector-seo/dot-device-selector-seo.component.html

```diff
@@ -0,0 +1,89 @@
+<p-overlayPanel #op styleClass="dotDeviceSelector__overlayPanel">
+    <div class="dotDeviceSelector__column">
```

Comment:
Add this new component to storybook please. We need to disccuss order tho, let's bring that discussion to the fe channel.